index.html

```diff
@@ -162,26 +162,38 @@ <h2>Introduction</h2>
   <p><a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a> currently support flat, unstructured strings for accessible names and descriptions.  The result of the name/description computation is thus a flat string. </p>
   <p>The terms "accessible name" and "accessible description" are used to emphasize that they are properties of <a class="termref">accessible objects</a> as exposed by <a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a>.  However, they are frequently referred to hereafter as simply "name" and "description".</p>
 </section>
+<section class="informative" id="terms">
+	<h1>Important Terms</h1>
+	<div>
+		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
+		<dl class="termlist">
+      <dt><dfn data-no>Tooltip attribute</dfn></dt>
+			<dd>
+			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents.</p>
```

Comment:
```suggestion
			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents (e.g., [[HTML]]'s [^html-global/title^]) attribute.</p>
```

Reply:
This needs to be a different issue and PR

Reply:
Likewise, can do that as a followup.